### PR TITLE
kraken: Relation frame route stop point

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: source/type/data.cpp [hunk 1]
```diff
@@ -426,16 +426,30 @@ void Data::build_associated_calendar() {
     }
 }
 
+
+static void build_frames(navitia::type::VehicleJourney* vj){
```
Comment (antoine-de) [inline source/type/data.cpp]: can you add a comment there to explain that this method builds all the frames' back references ?

File: source/type/pb_converter.cpp [hunk 2]
```diff
@@ -1146,7 +1146,8 @@ void PbCreator::Filler::fill_pb_object(const VjStopTimes* vj_stoptimes,
 
     pbnavitia::Uris* uris = pt_display_info->mutable_uris();
     uris->set_vehicle_journey(vj_stoptimes->vj->uri);
-    if (vj_stoptimes->vj->frame && vj_stoptimes->vj->frame->contributor){
+    if (vj_stoptimes->vj->frame && vj_stoptimes->vj->frame->contributor
+        && (!vj_stoptimes->vj->frame->contributor->license.empty())){
```
Comment (antoine-de) [inline source/type/pb_converter.cpp]: can you add a comment to explain the empty licence filtering ?

File: source/type/data.cpp (hunk 1 above)
Comment (azime) [inline source/type/data.cpp]: OK

File: source/type/pb_converter.cpp (hunk 2 above)
Comment (azime) [inline source/type/pb_converter.cpp]: OK

**Pull request merged**